### PR TITLE
More Bricks

### PR DESCRIPTION
**Pull request by broscolotos**
- added two variations of 4 different brick textures

### CODE REVIEW

**Review by RealTBEA — CHANGES_REQUESTED** [review]
you may want to doublecheck mortarBrick2's max metadata

File: src/main/java/com/bidahochi/BlockMod/core/handler/BlockIDs.java [hunk 1]
```diff
@@ -78,7 +78,8 @@ public enum BlockIDs
 
     //bricks
     fireBrick(false, null, "fireBrick"),
-    mortarBrick(true, ItemBlockmortarBrick.class, "mortarBrick", 8), // 8
+    mortarBrick(true, ItemBlockmortarBrick.class, "mortarBrick", 15),
+    mortarBrick2(true, ItemBlockmortarBrick2.class, "mortarBrick2", 1),
```
Comment (RealTBEA) [inline src/main/java/com/bidahochi/BlockMod/core/handler/BlockIDs.java:82]: ``

Comment (RealTBEA) [inline src/main/java/com/bidahochi/BlockMod/core/handler/BlockIDs.java:82]: `mortarBrick2(true, ItemBlockmortarBrick2.class, "mortarBrick2", 0)`

**Comment by broscolotos** [comment]
> I think you may want to double-check mortarBrick2's max metadata. I believe it should be 0 or not have the max metadta listed at all in the enum

true, lemme fix that

**Pull request merged**